### PR TITLE
Issue #33

### PR DESCRIPTION
**Pull request by rngadam**
Issue #33: remove autopublish and insecure

### CODE REVIEW

File: app/server/api/users.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Meteor.methods({
```
Comment (thimovss) [inline app/server/api/users.js:5]: Requires proper testing before execution

**Comment by rngadam** [comment]
added checking, now with automated tests:

screen shot 2016-09-01 at 2 10 02 pm

File: app/imports/server/api/users.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  it('accepts correct insert', function () {
+    Meteor.call('users.insert', {
+      profile: {
+        name: 'Ricky Ng-Adam', 
+        email: 'rngadam@example.com'
+      },
+      createdAt: new Date()
+    });
+  });
+
```
Comment (thimovss) [inline app/imports/server/api/users.test.js:50]: inconsistency with object syntax.

File: app/imports/server/api/users.test.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    expect = chai.expect;
+
+const VALID_DOC = {
+      profile: {
+        name: 'Ricky Ng-Adam', 
+        email: 'rngadam@example.com'
+      },
+      createdAt: new Date()
+    };
+
```
Comment (thimovss) [inline app/imports/server/api/users.test.js:23]: add test to confirm VALID_DOC remains valid ( maybe the schema changes later on)

File: README.md [hunk 4]
```diff
@@ -37,9 +37,16 @@ meteor run --settings ../secrets/settings/localhost-3000.json
...
+## Testing
+
+see: https://guide.meteor.com/testing.html
+
+Running tests:
 
 ```
-mongodump -h 127.0.0.1 --port 3001 -d meteor --collection users --out ../secrets/dump
-mongodump -h 127.0.0.1 --port 3001 -d meteor --collection UserProfile --out ../secrets/dump
+meteor test --driver-package practicalmeteor:mocha --port 3100
```
Comment (thimovss) [inline README.md:51]: not needed for now but maybe add `--settings` label.

**Comment by thimovss** [comment]
mail-password sign in added but there is no way to set a password and account creation is forbidden.

**Pull request merged**